### DISCUSSION TITLE
Copilot seat can not be removed on organisation

### DISCUSSION THREAD

**Discussion by MariusE** [community/community#62270] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi Community,
I manage an organization on Github and we added some Copilot-seats for testing.
Now I want to remove seats for a specific user by clicking the 'remove' button in the UI. This does not have any expected effect.

I checked the devtools for the request, sent when clicking the button. And that request seems to be ok. I used removed user-data here:
```
DELETE | https://github.com/organizations/{ORG_NAME}/settings/copilot/remove_seat?sort=name_asc&type=all&query=&page=1&user_id=xxxxxx

```

I also get a `200 OK`-response back, but that response just includes the seat-list like before the `DELETE`-reuqest, with `pending_cancellation_date: null` for all seats.

Is this a known issue, I just couldn't find?

**Comment by Matthewsre** [comment]
I also added a seat, but there is no way for me to remove it. I don't even see a button to remove.

**Comment by samus-aran** [comment]
Hi @MariusE & @Matthewsre 👋 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
